docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -28,23 +28,68 @@ videoBanner: UFhT52d5bYg
 - (!docs/pages/includes/database-access/aws-auto-discovery-prerequisite.mdx!)
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Create a Teleport user
+## Step 1/6. Create a Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
-## Step 2/5. Create a Database Service configuration
+## Step 2/6. Create a Database Service configuration
 
 (!docs/pages/includes/database-access/token.mdx!)
 
 (!docs/pages/includes/database-access/alternative-methods-join.mdx!)
 
 (!docs/pages/includes/database-access/aws-create-db-configuration.mdx type="Redshift" matcher="redshift" protocol="postgres" example="redshift-cluster-1.abcdefghijklm.us-west-1.redshift.amazonaws.com:5438"!)
 
-## Step 3/5. Create an IAM policy for Teleport
+## Step 3/6. Create an IAM Role for user access (optional) 
+
+Redshift supports two methods of IAM authentication, both of which Teleport
+also supports.
+
+First is IAM authentication as an database user. In this method, the Teleport
+Database Service generates the temporary IAM authentication token for a
+database user that already exists in the Redshift database. If you choose to
+use this method, you can skip this step as no additional IAM roles are
+required.
+
+The second method is to authenticate as an IAM role. In this case, the Teleport
+Database Service assumes this IAM role to authenticate with Redshift. Redshift
+then maps the IAM role to a database username and creates this database user if
+it doesn't already exist in the database.
+
+Now create the AWS IAM role to provide user access to the Redshfit database. Go
```

Comment:
Should this say, "If you choose the second method, create..." instead of "Now create..."?